manifests/users.pp

```diff
@@ -149,15 +159,21 @@
       $splayarg = ""
     }
 
+    if $clamps::agent::pxp_mock_puppet {
+      $puppet_run = "echo {} | ${config_path}/opt/pxp-agent/modules/pxp-module-puppet run"
```

Comment:
Hmm, this no longer uses cached catalog. It'd need some way to store and retrieve the last report variables. I'll probably work on that tomorrow.